core/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java

```diff
@@ -1108,6 +1119,231 @@ public void testSuggestWithManyCandidates() throws InterruptedException, Executi
         // assertThat(total, lessThan(1000L)); // Takes many seconds without fix - just for debugging
     }
 
+    @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return Collections.singleton(DummyTemplatePlugin.class);
+    }
+
+    public static class DummyTemplatePlugin extends Plugin implements ScriptPlugin {
+        @Override
+        public ScriptEngineService getScriptEngineService(Settings settings) {
+            return new DummyTemplateScriptEngine();
+        }
+    }
+
+    public static class DummyTemplateScriptEngine implements ScriptEngineService {
+
+        public static final String NAME = "mustache";
```

Comment:
The collate query setter is hard coded to use mustache, so I'm lying in this test about the script plugin, which makes the collate code thinks mustache is evaluating the query.